azure-pipelines/plato/generate-up-to-date-report.yml

```diff
@@ -0,0 +1,69 @@
+name: $(Date:yyyyMMdd).$(Hours)$(Minutes)$(Seconds)
+variables:
+  AZURE_STORAGE_ACCOUNT: vottv2
+trigger:
+  - dev*
+  - v2
+pr: none # disable CI build for PR
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+steps:
+- bash: |
+    set -e
+
+    COMMIT_SHA=`echo ${BUILD_SOURCEVERSION} | cut -c1-8`
+    echo "sha: " $COMMIT_SHA
+
+    # rewrite build number
+    echo "##vso[build.updatebuildnumber]Report-${COMMIT_SHA}-${BUILD_BUILDNUMBER}"
+  displayName: "Rewrite build number"
+
+- bash: |
+    set -e
+
+    ACCOUNT=$(AZURE_STORAGE_ACCOUNT)
+    if [[ -z "${ACCOUNT}" ]]; then
+      echo "Need to set AZURE_STORAGE_ACCOUNT"
+       exit 1
+    fi
+
+    KEY=$(AZURE_STORAGE_KEY)
+    if [[ -z "${KEY}" ]]; then
+      echo "Need to set AZURE_STORAGE_KEY"
+      exit 1
+    fi
+  displayName: "Verify storage account cred exists"
+
+- task: NodeTool@0
+  displayName: "Use Node 10.x"
+  inputs:
+    versionSpec: 10.x
+
+- task: Npm@1
+  displayName: 'Run `npm ci`'
+  inputs:
+    command: custom
+    verbose: false
+    customCommand: ci
+
+- task: AzureCLI@1
+  displayName: "Pull down old report and add updates"
+  inputs:
+    azureSubscription: 'PELITTLE TEAM - CSE DWR (d36d0808-a967-4f73-9fdc-32ea232fc81d)'
+    scriptLocation: inlineScript
+    inlineScript: './scripts/update-report.sh'
+
+- bash: |
+    set -e
+
+    cat /tmp/download.log
+  displayName: "print download log"
+  condition: succeededOrFailed()
+
+- task: PublishBuildArtifacts@1
```

Comment:
is this the new copy task that pushes to blob?

Reply:
no, this is just to publish the report as an artifact, which is then use in this release to copy to blob

https://dev.azure.com/msft-vott/VoTT/_releaseProgress?_a=release-environment-logs&releaseId=167&environmentId=167